packages/spektate/src/repository/IGitHub.ts

```diff
@@ -51,14 +51,14 @@ export const getManifestSyncState = (
               accessToken
             );
 
-            if (syncStatus != null) {
+            if (syncStatus != null && syncStatus.data && syncStatus.data.tag) {
```

Comment:
Negative tests for here too 
<img width="412" alt="image" src="https://user-images.githubusercontent.com/3119513/79785164-41766580-82f8-11ea-94ec-ba80f30af598.png">